classes/class-wc-connect-shipping-label.php

```diff
@@ -224,42 +242,42 @@ protected function get_form_schema() {
 				'properties' => array(
 					'height' => array(
 						'type' => 'number',
-						'title' => 'Height',
+						'title' => __( 'Height', 'woocommerce' ),
 					),
 					'product_id' => array(
 						'type' => 'string',
-						'title' => 'Product ID',
+						'title' => __( 'Product ID', 'woocommerce' ),
 					),
 					'length' => array(
 						'type' => 'number',
-						'title' => 'Length',
+						'title' => __( 'Length', 'woocommerce' ),
 					),
 					'quantity' => array(
 						'type' => 'number',
-						'title' => 'Quantity',
+						'title' => __( 'Quantity', 'woocommerce' ),
 					),
 					'weight' => array(
 						'type' => 'number',
-						'title' => 'Weight',
+						'title' => __( 'Weight', 'woocommerce' ),
 					),
 					'width' => array(
 						'type' => 'number',
-						'title' => 'Width',
+						'title' => __( 'Width', 'woocommerce' ),
 					),
 				),
 			);
 
 			$properties[ 'cart' ] = array(
 				'type' => 'array',
-				'title' => 'Items to send',
+				'title' => __( 'Items to send', 'woocommerce' ),
```

Comment:
Since it needs to know also about the contents of the package, I believed that the "Shopping Cart" concept was a good fit here. The way I see it, the merchant will be phisically putting the items into the boxes while he looks at that step of the form (and weighting them), so the list of items that must be inside each box is important too.